system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -302,12 +302,28 @@ int LdapUtils::getServerInfo(const char* ldapserver, const char* userDN, const c
             ld = ldapInitAndSimpleBind(ldapserver, nullptr, nullptr, "ldap", 389, cipherSuite, timeout, &err);
         }
 
-        // for new versions of openldap, version 2.2.*
-        if(nullptr == ld   &&  err == LDAP_PROTOCOL_ERROR  &&  stype != ACTIVE_DIRECTORY)
-            DBGLOG("If you're trying to connect to an OpenLdap server, make sure you have \"allow bind_v2\" enabled in slapd.conf");
+        if (ld == nullptr)
+        {
+            if (err == LDAP_PROTOCOL_ERROR && stype != ACTIVE_DIRECTORY)
+            {
+                DBGLOG("Unable to connect. If you're trying to connect to an OpenLdap server, make sure you have \"allow bind_v2\" enabled in slapd.conf");
```

Comment:
DBBLOG will not be seen in some systems that are defaulting to logging level of 55 (Progress level messages and above only).
I suspect this/others should be a WARNLOG's

Reply:
I was not comfortable with DBGLOG for the same reason, but was unsure what to use. Thanks for the suggestion.